### PR TITLE
Rb tabbed interface

### PR DESCRIPTION
**Pull request by Cindellai**

### CODE REVIEW

**Review by ghost — APPROVED** [review]
Thumbs up emoji 👍

File: app/views/users/liked.html.erb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    <ul class="nav nav-pills nav-justified">
+      <li class="nav-item">
+        <%= link_to "Posts", user_path(@user.username), class: "nav-link" %>
+      </li>
+      <li class="nav-item">
+        <%= link_to "Liked photos", liked_path(@user.username), class: "nav-link" %>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="#">Feed</a>
```
Comment (JoseMDO) [inline app/views/users/liked.html.erb:17]: We can use link_to methods here now :)

File: config/routes.rb [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
   resources :comments
   resources :follow_requests
   resources :likes
   resources :photos
+  
+  get ":username/liked" => "users#liked", as: :liked
+  get ":username/feed"
+  get ":username/followers"
+  get ":username/following"
+  get "/:username" => "users#show", as: :user
```
Comment (JoseMDO) [inline config/routes.rb:16]: Routes look good :)

**Review by JoseMDO — APPROVED** [review]
Looks good Chinemelum, great job!

**Pull request merged**